.github/workflows/tf-validate-ci.yml

```diff
@@ -0,0 +1,30 @@
+name: tf-validate-ci
+on:
+  push:
```

Comment:
do we want it on `push` or `pr`?
https://github.com/GoogleCloudPlatform/anthos-samples/blob/main/.github/workflows/ci_any_pr.yaml#L18

Reply:
ahh i see - would it be redundant, since we can only push onto main if the PR is up to date with main?